app/__init__.py

```diff
@@ -0,0 +1,25 @@
+"""The entry point for app.py to start our app
+"""
+from api import api
+from flask import Flask
+
+
+def create_app() -> Flask:
+
+    flask_app = Flask(__name__)
+    api.register(flask_app)
+    api.config.title = "Round Store API"
```

Comment:
Nice central configuration for the api. very good, should be easy to migrate this if needed in the future. (You could even read from a pyproject.toml for some of this info 👀 )

Reply:
Nice idea 👍  []